flex-squid/src/main/java/org/sonar/flex/metrics/FileLinesVisitor.java

```diff
@@ -45,9 +45,9 @@
 
   @Override
   public void visitFile(@Nullable AstNode astNode) {
-    linesOfCode = new HashSet<>();
-    linesOfComments = new HashSet<>();
-    noSonarLines = new HashSet<>();
+    linesOfCode.clear();
```

Comment:
@amelie-renard-sonarsource FYI I kept the `clear()` in `visitFile` to reset structure just to avoid misuse in case someone uses the same instance for multiple files, it is not the case currently (I checked its usages) but still better keep it.